### ISSUE TITLE
Add iPhone configurations to continuous integration

### ISSUE THREAD

**Issue by briansmith** [briansmith/ring#10]
64-bit at least, probably 32-bit too.

**Comment by briansmith** [comment]
32-bit iOS is no longer a thing.

`aarch64-apple-ios` is now built in CI/CD but the tests are not run. This is blocked on having M1 Mac runners. My understanding is that on M1 Macs, Xcode's iOS emulator will run ARM iOS executables.

https://medium.com/snips-ai/dinghy-painless-rust-tests-and-benches-on-ios-and-android-c9f94f81d305 notes that (prior to *ring* 0.16.17), [cargo dinghy](https://github.com/sonos/dinghy) had trouble building *ring*. I think that was because they were building *ring* from GitHub and so the warnings that were fixed in PR #1071 were probably breaking their build. So, while the aforementioned CI/CD job will probably help ensure *ring* keeps working with cargo dinghy, it might be worth having a new CI/CD job for that.

**Comment by briansmith** [comment]
Closing this in favor of #184, which has more history.
